### PR TITLE
added back button, changed position of warning and added button wrapp…

### PR DESCRIPTION
**Pull request by Mi-Lan**
…er border

### CODE REVIEW

File: app/src/theme/index.js [hunk 1]
```diff
@@ -112,8 +112,8 @@ const theme = {
     border: '1px solid #EEE',
     borderRadius: '8px ',
     boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.05)',
-    paddingHorizontal: '25px',
-    paddingVertical: '25px',
+    paddingHorizontal: '24px',
+    paddingVertical: '20px',
     textColor: '#000',
```
Comment (liaojikunwork) [inline app/src/theme/index.js:117]: @Mi-Lan , if you change here, all paddings of cards will be changed.
Is it ok? @pimato

**Review by liaojikunwork — CHANGES_REQUESTED** [review]
Please check comments

**Comment by pimato** [comment]
something is breaking here:
Bildschirmfoto 2020-11-06 um 16 42 44
just go to:
http://localhost:3000/#/0x06c942fd737bc53571a34667b2ae3b669f1f502d

**Review by pimato — CHANGES_REQUESTED** [review]
see comment

**Comment by Mi-Lan** [comment]
> something is breaking here:
> Bildschirmfoto 2020-11-06 um 16 42 44
> just go to:
> http://localhost:3000/#/0x06c942fd737bc53571a34667b2ae3b669f1f502d

@pimato hmmm i can't replicate that for me it works...
Screenshot 2020-11-06 at 19 30 26

File: app/src/theme/index.js (hunk 1 above)
Comment (Mi-Lan) [reply app/src/theme/index.js:117]: @liaojikunwork @pimato i mean i can overwrite it on component but kinda defeats the purpose of global styles if they are linked here they should be linked everywhere if we want to keep designs consistent...

Comment (pimato) [reply app/src/theme/index.js:117]: I agree with @Mi-Lan our goal is to get consistent from the design specification to the actual implementation. One issue I see now is that the navigation needs to be adopted according to my designs as part of it is not using the margin-top here:

Bildschirmfoto 2020-11-07 um 10 57 49

but I think this should be part of a new issue and PR!

**Comment by pimato** [comment]
> > something is breaking here:
> > Bildschirmfoto 2020-11-06 um 16 42 44
> > just go to:
> > http://localhost:3000/#/0x06c942fd737bc53571a34667b2ae3b669f1f502d
> 
> @pimato hmmm i can't replicate that for me it works...
> Screenshot 2020-11-06 at 19 30 26

After cloning the whole project everything works fine now!

**Comment by pimato** [comment]
the set allowance component should always be above the warning message:
Bildschirmfoto 2020-11-07 um 11 02 12

**Review by pimato — APPROVED** [review]

**Review by pimato — CHANGES_REQUESTED** [review]
see comment

**Review by pimato — APPROVED** [review]

**Review by liaojikunwork — APPROVED** [review]

File: app/src/components/market/common/set_allowance/index.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ const Wrapper = styled.div`
   border-radius: 4px;
   border: ${({ theme }) => theme.borders.borderLineDisabled};
   padding: 21px 25px;
+  margin-bottom: 20px;
```
Comment (KadenZipfel) [inline app/src/components/market/common/set_allowance/index.tsx:11]: Setting the `margin-bottom` of all `SetAllowance` components here causes an unexpectedly large margin in some places, e.g. 
image

Instead you should be including this in a style prop where this component exists in the component which you want to see the change

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

**Review by KadenZipfel — APPROVED** [review]

**Comment by pimato** [comment]
something is broken now:
Bildschirmfoto 2020-11-10 um 23 02 27

**Review by pimato — CHANGES_REQUESTED** [review]
see comments

**Comment by Mi-Lan** [comment]
> something is broken now:
> Bildschirmfoto 2020-11-10 um 23 02 27

fck i messed up that conflict will fix it asap dunno how i didn't catch it...

**Review by pimato — APPROVED** [review]

**Pull request merged**